conf/modules.config

```
@@ -174,3 +171,56 @@ params {
         }
     }
 }
+
+// nf-core modules DSL2 syntax v2.0
+process {
+
+    // publishDir = [
+    //     path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
+    //     mode: params.publish_dir_mode,
+    //     saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+    //]
+
+    withName: METABAT2_JGISUMMARIZEBAMCONTIGDEPTHS {
+        publishDir = [
+            path: { "${params.outdir}/GenomeBinning/" },
+            mode: 'copy',
+            pattern: '*-depth.txt.gz'
+        ]
+        ext.prefix = { "${meta.assembler}-${meta.id}-depth" }
+    }
+
+    withName: METABAT2_METABAT2 {
+        publishDir = [
+            [
+                path: { "${params.outdir}/GenomeBinning/MetaBAT2/" },
+                mode: 'copy',
+                pattern: 'bins/*.fa.gz'
+            ],
+            [
+                path: { "${params.outdir}/GenomeBinning/MetaBAT2/discarded_contigs" },
+                mode: 'copy',
+                pattern: '*tooShort.fa.gz'
+            ],
+            [
+                path: { "${params.outdir}/GenomeBinning/MetaBAT2/discarded_contigs" },
+                mode: 'copy',
+                pattern: '*lowDepth.fa.gz'
+            ]
```

To keep it in line with the output docs and previous versions:
```suggestion
            [
                path: { "${params.outdir}/GenomeBinning/MetaBAT2/discarded" },
                mode: 'copy',
                pattern: '*tooShort.fa.gz'
            ],
            [
                path: { "${params.outdir}/GenomeBinning/MetaBAT2/discarded" },
                mode: 'copy',
                pattern: '*lowDepth.fa.gz'
            ]
```

---

Ok, I'll try this again for you. Like I said in the opening PR I had at some points some issues where two modules seemed to overwrite the output of each other when they saved to the same publishDir (which is why I changed the folder structure slightly), but maybe that was a conflict between syntaxes.